missing deps

## Missing `yaml` dependency in @levino/shipyard-blog

  ### Description

  The `@levino/shipyard-blog` package imports `yaml` but doesn't declare it as a dependency in `package.json`. This causes builds to fail in clean environments (e.g., Docker) where transitive
  dependencies aren't available.

  ### Error

  Cannot find module 'yaml' imported from '/app/node_modules/@levino/shipyard-blog/src/index.ts'

  ### Affected files

  - `src/index.ts` - `import { parse as parseYaml } from 'yaml'`
  - `src/tags.ts` - `import { parse as parseYaml } from 'yaml'`

  ### Current dependencies in package.json

  ```json
  "dependencies": {
    "@levino/shipyard-base": "^0.7.0",
    "ramda": "^0.31"
  }

  Expected

  yaml should be listed in dependencies:

  "dependencies": {
    "@levino/shipyard-base": "^0.7.0",
    "ramda": "^0.31",
    "yaml": "^2.0.0"
  }

  Workaround

  Add yaml as an explicit dependency in the consuming project.
  ```

--- a/.changeset/add-yaml-dependency.md
+++ b/.changeset/add-yaml-dependency.md
@@ -1,0 +1,5 @@
+---
+'@levino/shipyard-blog': patch
+---
+
+The `yaml` dependency is now correctly declared in package.json. This fixes build failures in clean environments (e.g., Docker) where transitive dependencies aren't available.

--- a/packages/blog/package.json
+++ b/packages/blog/package.json
@@ -32,7 +32,8 @@
   "homepage": "https://shipyard.levinkeller.de",
   "dependencies": {
     "@levino/shipyard-base": "^0.7.0",
-    "ramda": "^0.31"
+    "ramda": "^0.31",
+    "yaml": "^2.0.0"
   },
   "peerDependencies": {
     "astro": "^5.15"

